packages/typescript/ai-memory/skills/tanstack-ai-memory-redis/SKILL.md

```
@@ -0,0 +1,52 @@
+await redis.connect()
+
+const memory = redisMemoryAdapter({ redis, prefix: 'myapp:memory' })
+
+memoryMiddleware({ adapter: memory, scope })
+```
+
+The adapter accepts any client implementing the `RedisLike` shape (a small subset: `get`, `set`, `del`, `sadd`, `srem`, `smembers`, `mget`). Both `redis` (node-redis v4+) and `ioredis` work.
+
+## Storage model
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)